invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposits/RDMDepositResults.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2020 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import {
+  LayoutSwitcher,
+  Pagination,
+  ResultsMultiLayout,
+  Sort,
+} from "react-searchkit";
+import { Button, Input, Checkbox, Grid, Segment } from "semantic-ui-react";
+
+export class Results extends Component {
+  render() {
+    const { sortValues, currentResultsState } = this.props;
+    const { total } = currentResultsState.data;
+    return (
+      total && (
+        <>
+          <Input icon="search" placeholder="Search uploads..." />
+          <Button
+            color="green"
+            icon="upload"
+            floated="right"
+            href="/deposits/new"
```

Comment:
This URL is hardcoded, I don't know if there's a cleaner way to inject flask URLs (similar to `url_for()`).